cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -337,6 +337,12 @@ describe('Dashboard', () => {
                   })
                 cy.getByTestID('save-cell--button').click()
 
+                // Make sure typeAhead input box is rendered and is visible
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${bucketVarName}`,
+                  {timeout: 10000}
```

Comment:
I think this is the default timeout, right? 10 seconds

Reply:
that's correct

Reply:
wait, why are we manually setting a timeout?

Reply:
`defaultCommandTimeout` is 4s but somehow asserting should before it's visible is breaking

Reply:
Remove it?